CHANGES.md

```diff
@@ -14,6 +14,7 @@
 - Fix `match`/`case` statements that contain `match`/`case` soft keywords multiple
   times, like `match re.match()` (#2661)
 - Fix `case` statements with an inline body (#2665)
+- Fix styling of starred expressions inside `match` subject (#2666)
```

Comment:
not sure if these are typically put in a specific order, from lowest pr to highest pr, as a result this would go below the next one

Reply:
It honestly doesn't matter. We put 'em in no real order.